templates/wizards/projects/asn1acn/createFilesList.cmake

```diff
@@ -38,11 +38,12 @@ function(tryCreateFiles fileNames)
 endfunction()
 
 function(appendPersistentListC)
-    set(persistentFiles ${ASN1SCC_C_DIR}/asn1crt.c ${ASN1SCC_C_DIR}/real.c)
+    set(persistentFiles ${ASN1SCC_C_DIR}/asn1crt.c)
 
     list(FIND ASN1SCC_C_OPTIONS --acn-enc ACN_ENCODING_IDX)
     if (NOT ${ACN_ENCODING_IDX} EQUAL -1)
-        set(persistentFiles ${persistentFiles} ${ASN1SCC_C_DIR}/acn.c)
+        set(persistentFiles ${persistentFiles} ${ASN1SCC_C_DIR}/asn1crt_encoding.c
+            ${ASN1SCC_C_DIR}/asn1crt_encoding_acn.c ${ASN1SCC_C_DIR}/asn1crt_encoding_uper.c)
```

Comment:
asn1crt_encoding - shouldn't that be added always?

why adding UPER encoding, when ACN was selected?